meeting_minutes/220811_SARIF_TC_64.md

```diff
@@ -0,0 +1,191 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Voting Member           |
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Gerald     | Sullivan  | Micro Focus       | Member                  |
```

Comment:
Gerald -> Jerry

Reply:
The registered name is Gerald and this is what we note here.